pipeline-runner/init.R

```diff
@@ -337,6 +339,14 @@ init <- function() {
             quit('no')
         }
 
+        # parse data from state machine input
+        input <- RJSONIO::fromJSON(input_json, simplify = FALSE)
+
+        # save logs to file
+        debug_subdir <- file.path(input$experimentId, debug_timestamp)
+        debug_path <- file.path("/debug", debug_subdir)
+        flog.appender(appender.tee(file.path(debug_path, "logs.txt")))
```

Comment:
what's this `flog.appender` doing?

Reply:
that line specifies a file to write logs to in addition to the console